safe-global/safe-wallet-web
# Remove guard flow

## What it solves

Part of #2067 

## How this PR fixes it

Moves the Remove guard flow to the new tx flow structure

## How to test it

1. Open a safe with a transaction guard
2. Remove the transaction guard
3. Observe the new layout

## Screenshots
<img width="1123" alt="Screenshot 2023-06-13 at 16 59 05" src="https://github.com/safe-global/safe-wallet-web/assets/5880855/ce0eae01-78f8-4f25-88b4-b5ceaf427ecb">

## Checklist
